src/features/player/index.ts

```diff
@@ -79,6 +79,7 @@ class Player {
     this.synths?.forEach((synth) => {
       synth?.setMasterVolume(vol)
     })
+    this.volume = vol
```

Comment:
On quick inspection, it looks like `this.volume` might be vestigial. At least, I couldn't find it accessed anywhere. Feel free to delete the property entirely from `Player`